modules/planning/tasks/optimizers/path_time_heuristic/gridded_path_time_graph.cc

```diff
@@ -143,6 +143,12 @@ Status GriddedPathTimeGraph::Search(SpeedData* const speed_data) {
 Status GriddedPathTimeGraph::InitCostTable() {
   // Time dimension is homogeneous while Spatial dimension has two resolutions,
   // dense and sparse with dense resolution coming first in the spatial horizon
+  constexpr double kDoubleEpsilon = 1.0e-6;  // The least meaningful unit_t_
+  if (unit_t_ < kDoubleEpsilon) {
+    // Sanity check for numerical stability
+    const std::string msg = "unit_t is smaller than the kDoubleEpsilon.";
+    AERROR << msg;
```

Comment:
If not reused, put the string after AERROR directly.

Reply:
Fixed. Thank you!